torch/_inductor/decomposition.py

```diff
@@ -460,16 +460,15 @@ def reshape_tensor_complex(tensor: torch.Tensor) -> torch.Tensor:
         reshaped_tensor = tensor.view(new_shape)
         return reshaped_tensor
 
+    x = aten.conj_physical(x)
+    z = aten.conj_physical(z)
+
     x_reshaped = reshape_tensor_complex(x.view(x.real.dtype))
     z_reshaped = reshape_tensor_complex(z.view(y.real.dtype))
     result = torch.flatten(x_reshaped + z_reshaped, start_dim=-2).view(complex_type)
     return result
 
 
-@register_decomposition([aten.conj_physical])
-def conj_physical(self: torch.Tensor) -> torch.Tensor:
-    assert not self.is_complex(), "TODO: implement this"
-    return self
```

Comment:
Rather than deleting this (which will regress models without complex tensors), can we do:
```py
if self.is_complex():
   return NotImplemented
```

Which will keep the op only if the input is complex.

Reply:
Thank you for the quick response, I have updated my change as suggested :)